app/models/container_image.rb

```diff
@@ -82,5 +82,14 @@ def has_compliance_policies?
     !plist.blank?
   end
 
+  def prevent_from_starting(causing_policy)
```

Comment:
@moolitayer please use the same name: `annotate_deny_execution`  (same for the rest in `miq_action`).